### DISCUSSION TITLE
Powering up the Jetson via Buck Converter

### DISCUSSION THREAD

**Discussion by JaneClelandHuang** [prl-mushr/mushr#51] [Hardware]
We are having a problem on some of our cars in powering the Jetson via the Buck converter.  We are passing exactly 5 volts to the Jetson but it only powers up about 10% of the time.  We've replaced Buck converters and it still has the problem.  The Jetsons all power up properly via the adapter.

**Comment by palanc** [comment]
Hi @JaneClelandHuang - this issue can be caused by the Jetson Nano's sensitivity to power supply during booting. Two things to try:

1. **Increase the buck converter output voltage to 5.2V.** The initial surge of power required to boot the Jetson can cause the buck converter's output voltage to momentarily drop. If this drop causes the output voltage to fall below approximately 4.75V, the Nano won't boot. Raising the nominal output voltage of the buck converter to 5.2V is still within the Nano's recommended input voltage, but provides additional voltage margin in the case of transient power surges.  

2. **Make sure that the initial contact between the battery cable and the buck converter input cable is stable rather than intermittent.** This can be helped by straightening the portions of wire that are near the male and female receptacles being connected, as well as decisively pushing the two cable receptacles together when making the connection.

Let us know if either of those suggestions help.

**Reply by JaneClelandHuang** [reply]
Unfortunately, we've tried both things previously -- increasing the voltage up to 5.5V, and straightening the wires.  Have you had these problems too.

**Comment by palanc** [comment]
We have seen your original problem before, but it was always resolved by one or both of those suggestions. 

When increasing the output voltage, did you jump directly to 5.5V, or did you take intermittent steps (particularly 5.2V)? I ask because if you supply **too** much voltage, then the Nano's overvoltage protection circuit can also prevent it from booting.

Also just want to make sure that the batteries were charged?

**Reply by JaneClelandHuang** [reply]
Yes batteries were charged :-).  I'm going to ask Raj (our car builder) to post to this thread.

**Comment by brightsnowzhao** [comment]
Want to contribute more data point: I had the same issue.  I solved it by tuning voltage to 5.4V and always pushing power plug in decisively.  If I push it slowly, it may still fail to boot.  The nano worked well on power supply, which I measured and found voltage of 5.4V.  Hence I replicated that voltage on the buck convertor.  I tried 5.2V and 5.3V, neither could work.
